web/src/main/java/org/cbioportal/web/GeneController.java

```diff
@@ -88,7 +88,7 @@
             @RequestBody List<String> geneIds) throws PageSizeTooBigException {
 
         if (geneIds.size() > PagingConstants.MAX_PAGE_SIZE) {
-            throw new PageSizeTooBigException(geneIds.size());
+            throw new PageSizeTooBigException(geneIds.size()); //TODO - is this correct? Missing API documentation on how one can specify page size parameter 
```

Comment:
My initial API design didn't have any Paging and Sorting capabilities, but I added them to the APIs I implemented during implementation, so we need to add the same capabilities to this API as well. Sorry that I forgot to mention.